__tests__/app.test.js

```diff
@@ -23,6 +23,29 @@ describe('GET /api/topics', () => {
     });
 });
 
+describe('GET /api', () => {
+    test('STATUS 200: reponds with an object describing all the available endpoints on the API', () => {
```

Comment:
I can completely see your thinking behind the testing but say for example you had an endpoint that you (legitimately) didn't have a queries property for then this test would fail. Feel free to require in the endpoints.json into the test file and assert it is equal to the one returned by the enpoint. This feels odd but you are mainly just interested the the endpoint serves that file. I hope this makes sense.